src/styles/index.scss

```diff
@@ -1,3 +1,143 @@
+@use "inc" as *;
+
 body {
   margin: 0;
+  background-color: #f3f3f3;
+  font-family: $bodyfont;
+}
+
+.card {
+  width: 100%;
+  max-width: rem(200);
+  border-radius: rem(5);
+  border: rem(1) solid #f3f3f3;
+  background: $white;
+  padding: rem(32) rem(16) rem(16) rem(16);
+  display: flex;
+  flex-direction: column;
+
+  &__featured {
+    width: 100%;
+    max-width: rem(160);
+    height: rem(134);
+    overflow: hidden;
+    margin: 0 auto rem(40) auto;
+
+    img {
+      width: 100%;
+      object-fit: contain;
+    }
+  }
+
+  &__title {
+    color: $main;
+    font-size: rem(12);
+    font-weight: 500;
+    line-height: lineheight(18, 12);
+    margin-bottom: rem(4);
+    text-decoration: none;
+  }
+
+  &__caption {
+    color: $second;
+    font-size: rem(10);
+    font-weight: 400;
+    line-height: lineheight(14, 10);
+    margin-bottom: rem(16);
+  }
+
+  &__reviews {
+    margin-bottom: rem(24);
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+
+    &-stars {
+      list-style: none;
+      display: flex;
+      align-items: center;
+      padding-left: 0;
+    }
+
+    $stars: 1, 2, 3, 4, 5;
+
+    @each $star in $stars {
+      &-stars--#{$star} {
+        .card__reviews-star:nth-child(-n + #{$star})::after {
+          background-image: url("../images/star-active.svg");
+        }
+      }
+    }
+
+    &-star {
+      position: relative;
+      width: rem(16);
+      height: rem(16);
+
+      &:not(:last-child) {
+        margin-right: rem(4);
+      }
+
+      &::after {
+        content: "";
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        background-position: center;
+        background-repeat: no-repeat;
+        background-image: url("../images/star.svg");
+      }
+    }
+
+    &-count {
+      font-size: rem(10);
+      color: $main;
+      font-weight: 400;
+      line-height: lineheight(14, 10);
+    }
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: rem(16);
+
+    &-caption {
+      font-size: rem(12);
+      color: $second;
+      line-height: lineheight(18, 12);
+      font-weight: 400;
+    }
+
+    &-qnt {
+      font-size: rem(16);
+      color: $main;
+      font-weight: 700;
+      line-height: lineheight(18, 16);
+    }
+  }
+
+  &__buy {
+    color: $white;
+    width: 100%;
+    height: rem(40);
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding instead